tests/data/annotations/mergeable/minion-annotations-differ.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     nginx.org/mergeable-ingress-type: "master"
     nginx.org/proxy-send-timeout: "10s"
     nginx.org/max-conns: "108"
+    nginx.org/proxy-set-headers: "X-Forwarded-ABC"
```

Comment:
@AlexFenlon lets add another test for standard ingress when annotation is applied to all the paths